docs/data/joy/components/modal/modal.md

```diff
@@ -129,7 +155,7 @@ const theme = extendTheme({
 });
 ```
 
-For **TypeScript**, you need module augmentation to include the new values to the `layout` prop:
+When using TypeScript, module augmentation is required to incorporate the new values into the layout property:
```

Comment:
We try to avoid using the word "property" when we're specifically referring to "React props"

```suggestion
When using TypeScript, module augmentation is required to incorporate the new values into the `layout` prop:
```